### PR TITLE
Vm-111: Testing

### PR DESCRIPTION
**Pull request by LihiniAbeykoon**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1 +1 @@
-console.log("Hello from Lihini!");
\ No newline at end of file
+console.log("Hello from Lihini! It's a new addition");
```
Comment (GangulalWijekoon) [inline app.js]: first review comment

Comment (GangulalWijekoon) [inline app.js:1]: first review comment

Comment (GangulalWijekoon) [inline app.js:1]: first review comment

Comment (GangulalWijekoon) [inline app.js:1]: first review comment

Comment (GangulalWijekoon) [inline app.js]: second review comment

Comment (GangulalWijekoon) [inline app.js:1]: second review comment

Comment (Rushabh01234) [reply app.js]: Test comment

Comment (LihiniAbeykoon) [inline app.js]: This is a new comment from Lihini!

Comment (Rushabh01234) [reply app.js:1]: Test comment 2

**Review by GangulalWijekoon — COMMENTED** [review]
new test review comment (3)

File: app.js (hunk 1 above)
Comment (GangulalWijekoon) [inline app.js]: review comment 4

File: app.js [hunk 2]
```diff
@@ -1 +1,2 @@
...
+console.log("Hello from Lihini! This for test");
```
Comment (GangulalWijekoon) [inline app.js:2]: comment for test db

Comment (LihiniAbeykoon) [inline app.js:2]: Test com

Comment (LihiniAbeykoon) [inline app.js:2]: Test com2

File: app.js [hunk 3]
```diff
@@ -1 +1,2 @@
-console.log("Hello from Lihini!");
\ No newline at end of file
+console.log("Hello from Lihini! It's a new addition");
```
Comment (LihiniAbeykoon) [inline app.js:1]: test com3

File: app.js [hunk 4]
```diff
@@ -1 +1,2 @@
-console.log("Hello from Lihini!");
```
Comment (LihiniAbeykoon) [inline app.js:1]: test com4

Comment (GangulalWijekoon) [inline app.js:1]: new review comment

Comment (GangulalWijekoon) [inline app.js:1]: test app

Comment (GangulalWijekoon) [inline app.js:1]: test app2

**Pull request merged**